schema/validator.go

```diff
@@ -251,3 +252,22 @@ func checkPlatform(OS string, Architecture string) {
 	}
 	fmt.Printf("warning: operating system %q of the bundle is not supported yet.\n", OS)
 }
+
+func validateArtifact(r io.Reader) error {
+	header := v1.Artifact{}
+
+	buf, err := ioutil.ReadAll(r)
```

Comment:
`ioutil` was deprecated in 1.16 so we're overdue to update this.

```suggestion
	buf, err := io.ReadAll(r)
```